src/libpgmoneta/prometheus.c

```diff
@@ -271,7 +271,7 @@ home_page(int client_fd)
    data = pgmoneta_append(data, "  </ul>\n");
    data = pgmoneta_append(data, "  <p>\n");
    data = pgmoneta_append(data, "  <h2>pgmoneta_retention</h2>\n");
-   data = pgmoneta_append(data, "  The retention of pgmoneta in days\n");
+   data = pgmoneta_append(data, "  The retention of pgmoneta in days, weeks, months and years\n");
```

Comment:
Write an entry for each

Reply:
I'm not so familiar with prometheus usages, can I define the header myself? like pgmoneta_retention_days, pgmoneta_retention_weeks...

Reply:
Yes, there needs to be a unique value for each